### PR TITLE
P2689R2 - SG1 Issaquah 2023 Changes

### PR DESCRIPTION
**Pull request by nliber**

### CODE REVIEW

File: atomic_accessor/atomic_accessor.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 title: "Atomic Refs Bounded to Memory Orderings & Atomic Accessors"
-document: P2689R1
+document: P2689R2
```
Comment (dalg24) [inline atomic_accessor/atomic_accessor.md:3]: Does the title need to be updated as well "Bound[ed]"

Comment (dalg24) [reply atomic_accessor/atomic_accessor.md:3]: (They are more occurrences that would need fixing)

File: atomic_accessor/atomic_accessor.md [hunk 2]
```diff
@@ -612,12 +654,11 @@ constexpr basic-atomic-accessor(basic-atomic-accessor<OtherElementType, Referenc
...
 constexpr reference access(data_handle_type p, size_t i) const noexcept;
 ```
-[2]{.pnum} *Effects:* Equivalent to `return reference(p + i);`
-
+[2]{.pnum} *Effects:* Equivalent to `return p[i];`
 ```c++
 constexpr data_handle_type offset(data_handle_type p, size_t i) const noexcept;
 ```
-[3]{.pnum} *Effects:* Equivalent to `return p[i];`
+[3]{.pnum} *Effects:* Equivalent to `return reference(p + i);`
```
Comment (dalg24) [inline atomic_accessor/atomic_accessor.md:681]: Ugggg not so sure about the "`reference(...)`" part

File: atomic_accessor/atomic_accessor.md (hunk 1 above)
Comment (nliber) [reply atomic_accessor/atomic_accessor.md:3]: Fixed (in a few places in the prose as well).

File: atomic_accessor/atomic_accessor.md (hunk 2 above)
Comment (nliber) [reply atomic_accessor/atomic_accessor.md:681]: Could you elaborate?  That is the only place the passed in reference type is used.

Comment (nliber) [reply atomic_accessor/atomic_accessor.md:681]: Fixed (I think).

Comment (dalg24) [reply atomic_accessor/atomic_accessor.md]: `offsets` returns `data_handle_type` (aka `ElementType*`) which is not constructible from `reference` (aka `atomic_ref<ElementType>` or alike)

**Review by dalg24 — APPROVED** [review]

File: atomic_accessor/atomic_accessor.md (hunk 2 above)
Comment (dalg24) [reply atomic_accessor/atomic_accessor.md]: (would have sworn you said "can you elaborate")
anyway looks good now

File: atomic_accessor/atomic_accessor.md [hunk 3]
```diff
@@ -20,6 +20,64 @@ toc: true
 
 # Revision History
 
+## P2689R2 (SG1 Issaquah 2023 discussion)
+
+- Renamed `atomic-ref-bounded` to `atomic.ref.bound`
```
Comment (crtrott) [inline atomic_accessor/atomic_accessor.md:25]: `atomic-ref-bound` ??

**Review by crtrott — APPROVED** [review]

**Pull request merged**